api/src/main/java/com/epam/pipeline/acl/datastorage/DataStorageApiService.java

```diff
@@ -416,4 +418,25 @@ public List<DataStorageTagSearchResult> searchDataStorageItemByTag(
             final DataStorageObjectSearchByTagRequest request) {
         return dataStorageManager.searchDataStorageItemByTag(request);
     }
+
```

Comment:
I'd suggest to have a separate set of controller-service-manager for this permissions as data storage classes are already too big

Reply:
done